app/templates/components/forms/reset-password-form.hbs

```diff
@@ -21,6 +21,11 @@
               <div class="ui left icon input">
                 <i class="key icon"></i>
                 {{input type='password' name='password' placeholder=(t 'password') value=password}}
+                  <div class="ui small basic icon buttons">
+                    <button type="button" class="ui button" {{action 'showPassword' 'New'}}>
```

Comment:
The showPassword button is not working. Have you defined the `showPassword` action?

Reply:
I had already committed it locally? The indentation and toogle icon action both